streams/src/main/java/org/apache/kafka/streams/processor/internals/tasks/TaskManager.java

```diff
@@ -121,4 +119,14 @@ public interface TaskManager {
      */
     Map<TaskId, StreamsException> drainUncaughtExceptions();
 
+    /**
+     * Signals that at least one task has become processable, e.g. because it was resumed or new records may be available.
```

Comment:
I assume this method with be used in later commits?

Reply:
Yes!